src/internal/analytics/components/analytics-funnel.tsx

```diff
@@ -162,6 +163,7 @@ const InnerAnalyticsFunnel = ({ children, stepConfiguration, ...props }: Analyti
         componentTheme: isVisualRefresh ? 'vr' : 'classic',
         funnelVersion: FUNNEL_VERSION,
         stepConfiguration: stepConfiguration ?? singleStepFlowStepConfiguration,
+        resourceType: props.funnelResourceType || getTextFromSelector(getBreadcrumbLinkSelector(3)),
```

Comment:
@michaeldowseza Based on my audit, I noticed second breadcrumb item represent the resourceType. https://quip-amazon.com/hAHwAOda7tDQ/Edit-flow-resource-type.

Any reason to pick third item here?